### ISSUE TITLE
Bootstrap and Bulma CSS options is not shown at rails new on 7.0.0.alpha2

### ISSUE THREAD

**Issue by edwinlunando** [rails/rails#43251]
### Steps to reproduce
Install the 7.0.0.alpha2 gem and try to run `rails new` to find out the new js and css options.
```
gem install rails -v 7.0.0.alpha2
rails new
```

It seems that the [current rails new generator](https://github.com/rails/rails/blob/ba57ad52453ef52b242af609d86be5705afde2fe/railties/lib/rails/generators/rails/app/app_generator.rb#L265) is not updated from the added [bootstrap](https://github.com/rails/cssbundling-rails/pull/3) and [bulma](https://github.com/rails/cssbundling-rails/pull/19) support. It would be beneficial to list all the options to provide more clarity. If yes, I am willing to fix this to make my first contribution to Rails.

### Expected behavior
The CSS options should give all the available options listed on the [css-bundling](https://github.com/rails/cssbundling-rails/) gem that includes `bootstrap` and `bulma`.

### Actual behavior
The `rails new` output docs did not state all the possible options for CSS. The missing options are `bootstrap` and `bulma`.
```
  -j, [--javascript=JAVASCRIPT]                              # Choose JavaScript approach [options: importmap (default), webpack, esbuild, rollup]
                                                             # Default: importmap
      [--css=CSS]                                            # Choose CSS processor [options: tailwind, postcss, sass]
```

### System configuration
**Rails version**: 7.0.0.alpha2

**Ruby version**: 3.0.0

**Comment by ceritium** [comment]
I think that it is due `7.0.0.alpha2` is released with `cssbundling-rails@0.1.0` which doesn't include `bootstrap` or `bulma`.

As a workaround, you can generate a new rails app without the CSS option, next add and follow the instructions of [cssbundling-rails](https://github.com/rails/cssbundling-rails#installation).

Notice that "Bulma" implementation is not released yet on "cssbundling, but you can point the gem to the main branch to get the last changes.

**Comment by ceritium** [comment]
**Update:**  As @dhh said here https://github.com/rails/rails/pull/43255, `rails new` will use the last version available of `cssbundling-rails`. A new release is available, so `bulma` and `bootstrap` generators should be working now.

https://github.com/rails/rails/pull/43254 points to the `cssbundling-rails` readme instead of updating the list every time there is a new addition.

You may consider closing this issue now or after merge the PR.
